docs/detections/rules-ui-monitor.asciidoc

```diff
@@ -69,6 +69,14 @@ You can also use Task Manager in {kib} to troubleshoot background tasks and proc
 * {kibana-ref}/task-manager-health-monitoring.html[Task Manager health monitoring]
 * {kibana-ref}/task-manager-troubleshooting.html[Task Manager troubleshooting]
 
+[float]
+[[troubleshoot-max-alerts]]
+==== Troubleshoot max alerts warning
+
+When a rule reaches the maximum number of alerts it can generate in a single rule execution, the following warning is displayed on the rule's details page and in the rule execution log: `This rule reached the maximum alert limit for the rule execution. Some alerts were not created.` 
+
+If you receive this warning, check the alerts and determine whether they should've been created. If any were wrongfully created, start investigating what caused the excess alerts. For example, check the rule's data source for possible issues or modify the rule's query if it's too broad in scope.
```

Comment:
> For example, check the rule's data source for possible issues or modify the rule's query if it's too broad in scope.

I think this is a good first step for triaging, @marshallmain brought up a good idea of referencing alert suppression for KQL rules and rule exceptions to limit alert volume as well.

Reply:
Yeah, it could be helpful to include tips for controlling alert volume as well -- especially if those tips apply to specific rule types. How about something like this:

If you receive this warning, go to the rule's **Alerts** tab and examine the alerts for anything unexpected. Unexpected alerts might be created from data source issues or queries that are too broadly-scoped. To further reduce alert volume, you can also add <<add-exceptions,rule exceptions>> or <<alert-suppression,suppress alerts>>.

Reply:
Yeah, it could be helpful to include tips for controlling alert volume as well -- especially if those tips apply to specific rule types. How about something like this:

```suggestion
If you receive this warning, go to the rule's **Alerts** tab and examine the alerts for anything unexpected. Unexpected alerts might be created from data source issues or queries that are too broadly-scoped. To further reduce alert volume, you can also add <<add-exceptions,rule exceptions>> or <<alert-suppression,suppress alerts>>. 
```

Reply:
Yeah, I think that reads well